gradle/dependencies.gradle

```diff
@@ -3,7 +3,7 @@ ext.versions = [
     dokka: '1.4.32',
     kotlinCoroutines: '1.5.0',
     compileSdk: 30,
-    idea: '201.8743.12',
+    idea: '202.8194.7',
```

Comment:
just double checking - latest Android Studio stable version is also on 2020.2?

Reply:
it's actually 2020.3.1 but it requires java 11 :/

Reply:
> like intellij platform apis on 2020.3.1 require java 11?

[yup](https://blog.jetbrains.com/platform/2020/09/intellij-project-migrates-to-java-11/)